### ISSUE TITLE
[Bug] No beta

### ISSUE THREAD

**Issue by jimfarrand** [ankidroid/Anki-Android#13083]
###### Reproduction Steps

I'm very interested in trying out the V3 scheduler, which I gather is a beta-only feature at the moment.  There are a couple of different sets of instructions for joining the beta program floating around the web.

One says that I should open a private ticket in the main Anki github project with some specific bits of information.  If did this and it was closed saying it was the wrong thing to do and that I should open one here instead.

The other says I should sign up via the Google Play Beta.  This says in turn that, after signing up I'll receive a beta version on my device automatically after a short delay.  I signed up more than a week ago but this hasn't happened.  Visiting the Google Play page for Ankidroid I don't see a way of forcing an update.  

###### Expected Result

My device upgrades to beta version with the V3 scheduler option, as per documentation.

###### Actual Result

No beta version.

###### Debug info

Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".

2.15.6

###### Research
*Enter an [x] character to confirm the points below:*

- [X] I have read the [support page]() and am reporting a bug or enhancement request specific to AnkiDroid
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] I have searched for similar existing issues here and on the user forum

- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

The instructions for Beta testing don't seem to work, and the instructions for Alpha testing are flagging "Permission denied".

**Comment by david-allison** [comment]
> One says that I should open a private ticket in the main Anki github project with some specific bits of information. If did this and it was closed saying it was the wrong thing to do and that I should open one here instead.

Where did you receive the information to post a ticket to the main Anki GitHub? That isn't correct and we should change it.

----

SchedV3 is in 2.16alpha, not yet in beta.

Google Play won't accept our alpha builds until we make changes. We're looking into resolving this. 

For now, you may find alpha builds/parallel builds: https://github.com/ankidroid/Anki-Android/releases/

Please note that these don't auto-update. Use alpha software at your own risk.

----

Closing this as resolved

**Comment by dae** [comment]
They originally posted on the AnkiMobile support site, and I redirected them here. I expected them to do a search rather than posting a bug, but I could have been clearer.

It might be nice if a note about the current alpha testing situation was added to https://docs.ankidroid.org/#betaTesting, as currently that info doesn't apply due to the scoped storage issue.

**Comment by jimfarrand** [comment]
> They originally posted on the AnkiMobile support site, and I redirected them here. I expected them to do a search rather than posting a bug, but I could have been clearer.

In my defence, I did do a a number of searches, and I, found the documentation you linked above, followed it, saw that nothing happened, waited a bit, backed up my AnkiDroid, uninstalled it, reinstalled it, followed the instructions again, and then did some more searching.  Only then did I raise a bug.  Which reasonable at the time as the instructions for installing the beta clearly weren't installing a beta and I had made a decent effort to unblock myself.  

I realise now that my Google search results conflate "ankidroid" and "ankimobile", which I now understand are rather different projects.  I didn't previously and so I didn't notice I'd ended up on the wrong site.

Obviously, I feel quite foolish now.  I do apologise for wasting everyone's time.  :)
